acstools/acscteforwardmodel.py

```diff
@@ -8,6 +8,8 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob/master/acs_cte_forward_model/acs_cte_forward_model_example.ipynb>`_.
```

Comment:
Nitpick, please break this long sentence into multiple lines, preferably to have line breaks every 80th char. :wink: